src/main/resources/db/migration/postgres/V1.0.0.0__base_acas_schema_tables.sql

```diff
@@ -2188,7 +2188,6 @@ REVOKE ALL ON SCHEMA acas FROM PUBLIC;
 REVOKE ALL ON SCHEMA acas FROM acas;
 GRANT ALL ON SCHEMA acas TO acas;
 GRANT ALL ON SCHEMA acas TO postgres;
-GRANT USAGE ON SCHEMA acas TO seurat;
```

Comment:
The ones below make sense as we are getting rid of the "compound" schema. This one seems unrelated, but I guess we don't need to build in native base support for a "seurat" user.